pkg/kubelet/prober/prober_manager.go

```diff
@@ -48,6 +48,22 @@ var ProberResults = metrics.NewCounterVec(
 		"pod_uid"},
 )
 
+// ProberDuration stores the duration of a successful probe lifecycle by result as prometheus metrics.
+var ProberDuration = metrics.NewHistogramVec(
+	&metrics.HistogramOpts{
+		Subsystem:      "prober",
+		Name:           "probe_duration",
+		Help:           "Duration in seconds for a probe response",
```

Comment:
I know that today the duration is in seconds, but we started discussions about sub-second duration. (for example: https://github.com/mikebrow/kubernetes/commit/8926bd2c021a63f365d1a89ed465bd2c5abe599b) Can we change this to milliseconds?

Reply:
The unit behind the histogram can be anything, but usually, for duration, seconds are preferred. It is then up to you to define buckets that better suit your needs. The default ones are:
```
.005, .01, .025, .05, .075, .1, .25, .5, .75, 1, 2.5, 5, 7.5, 10
```
So you'll be able to record any operation with duration down to 5 milliseconds.

Reply:
these buckets looks OK to me.